tom256/src/rng.rs

```diff
@@ -0,0 +1,23 @@
+use rand::rngs::{OsRng, StdRng};
+use rand_core::{CryptoRng, RngCore, SeedableRng};
+
+pub trait RngDefault {
+    fn default() -> Self;
+}
+
+pub trait CryptoCoreRng: RngCore + CryptoRng + RngDefault {}
+
+impl RngDefault for OsRng {
```

Comment:
Here every impl for `StdRng` could be added under the `#[cfg(test)]` flag because our lib does not explicitly depend on that Rng. If we build to the wasm32 target, then we need `OsRng`, but that can be imported directly from `rand_core`

Reply:
Unfortunately, we need `rand` as a normal dependency since we use it in the wasm generated functions for the attest proof. This also means that the traits must be compiled in all configurations as well.

Reply:
Nevermind, It works if OsRng is imported from rand_core. `StdRng` caused some problems but it was moved behind `#[cfg(test)]`.